src/items/functions.md

```diff
@@ -2,12 +2,12 @@
 
 > **<sup>Syntax</sup>**\
 > _Function_ :\
-> &nbsp;&nbsp; _FunctionFront_ `fn` [IDENTIFIER]&nbsp;[_Generics_]<sup>?</sup>\
+> &nbsp;&nbsp; _FunctionQualifiers_ `fn` [IDENTIFIER]&nbsp;[_Generics_]<sup>?</sup>\
 > &nbsp;&nbsp; &nbsp;&nbsp; `(` _FunctionParameters_<sup>?</sup> `)`\
 > &nbsp;&nbsp; &nbsp;&nbsp; _FunctionReturnType_<sup>?</sup> [_WhereClause_]<sup>?</sup>\
 > &nbsp;&nbsp; &nbsp;&nbsp; [_BlockExpression_]
 >
-> _FunctionFront_ :\
```

Comment:
This will need to be updated in a few other places (types, associated-items, traits).

Reply:
Done and rebased. My bad for not going a grep.

Reply:
Thanks, I think it's much better than my original.

The new update uses `_FunctionQualifier_` singular, whereas the definition is `_FunctionQualifiers_` plural.  I think they all should probably be the plural one.

Reply:
That was just me being tired in the morning. Fixed and rebased with just that change.